### PR TITLE
[MM-54745] Fixing inconsistent styles for menu items in profile/accou…

### PR DESCRIPTION
**Pull request by sapnasivakumar**
…nt menu
#### Summary
In the profile menu, the following style issues have been fixed :

1. Updated the font used for the display name to: font-family: 'Metropolis', sans-serif; font-size: 16px;
2. Updated color of the username to: rgba(var(--center-channel-color-rgb), 0.75);
3. Updated capitalization to use sentence case for menu items (The english versions have been updated in the corresponding i18n translation files, should discuss about an approach to update the rest of the languages)
4. Updated the default state of ‘Set a Custom Status’ menu item text to: rgba(var(--center-channel-color-rgb), 0.75);
5. The checkmark icon that appears on the current status has been
updated to: var(--button-bg);
6. Profile and Logout menu options have been 
updated to these icon colors : rgba(var(--center-channel-color-rgb), 0.56);

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/25263
Jira https://mattermost.atlassian.net/browse/MM-54745

#### Screenshots
The changes made match the proposed update below

Screenshot 2023-12-13 at 19 12 35

Screenshot 2023-12-13 at 19 13 36

#### Release Note
```release-note
Improved appearance of profile/account menu
```

### CODE REVIEW

**Review by matthewbirtch — CHANGES_REQUESTED** [review]
This is great @sapnasivakumar. A couple of minor tweaks:

1. For the 'set a custom status' text, I'd like to make the text full opacity of the `center-channel-color` so it doesn't look disabled. And let's do this for both the default state and for the the state when the custom status is actually set.
image

image

This should look like this:
image

image

2. For the hover state, can we make the icons for set custom status (default state), profile and logout to have use 'rgba(var(--center-channel-color-rgb), 0.72);` for the color of the icons on hover of the menu row.

**Comment by sapnasivakumar** [comment]
> 1. For the 'set a custom status' text, I'd like to make the text full opacity of the `center-channel-color` so it doesn't look disabled. And let's do this for both the default state and for the the state when the custom status is actually set.


Done ✅

**Comment by sapnasivakumar** [comment]
> 2. For the hover state, can we make the icons for set custom status (default state), profile and logout to have use `rgba(var(--center-channel-color-rgb), 0.72);` for the color of the icons on hover of the menu row.

Trying to change, Will update with a commit soon

**Review by matthewbirtch — APPROVED** [review]
@sapnasivakumar I'm going to be away for the next few weeks, so I'm going to approve this to unblock, but @asaadmahmood will finish off the ux review and help shepherd this through the rest of the review process (dev review and QA review still need to be done)

**Comment by asaadmahmood** [comment]
@sapnasivakumar Let me know once those changes are done and I'll rereview.

**Comment by sapnasivakumar** [comment]
@asaadmahmood will update sooner and let you know

**Comment by sapnasivakumar** [comment]
@asaadmahmood 

> 2. For the hover state, can we make the icons for set custom status (default state),

I gave it a check and this is already in place for the set custom status icon. 
Whereas for the logout and profile icons, changing the opacity on hovering over the icons is possible. But I observe that on hovering over the menu row if we change the style it might reflect across every other spot in the application using that style. 

I'm unaware of any other work around to get this going. 

I would like to know your thoughts on this.

**Comment by asaadmahmood** [comment]
@sapnasivakumar The color still doesn't look right.
Screenshot 2024-01-05 at 2 00 43 AM

**Review by asaadmahmood — APPROVED** [review]
LGTM!

File: webapp/channels/src/i18n/en_AU.json
Comment (hmhealey) [inline webapp/channels/src/i18n/en_AU.json:5047]: Could you actually revert the changes to this file? Our translation system tends to run into conflicts if files other than `en.json` are updated directly in the repo

**Comment by hmhealey** [comment]
Actually, could you also address the issues with the type checker and linter? You can run them locally by running `npm run check-types` and `npm run check` respectively.

Let me know if you need help with the type checker one since that's not an error that I've seen before

File: webapp/channels/src/i18n/en_AU.json
Comment (sapnasivakumar) [reply webapp/channels/src/i18n/en_AU.json]: Done 👍🏻

**Comment by matthewbirtch** [comment]
Hi @sapnasivakumar just checking in on this PR. Did you need any help to complete it?

**Comment by matthewbirtch** [comment]
Thanks @sapnasivakumar. 

@wiggin77 and @hmhealey could you review again when you have a chance?

**Comment by hmhealey** [comment]
@sapnasivakumar Would you mind updating the snapshots again? You updated those earlier, but they've changed again because of some changes you made since then.

Otherwise, the code for this looks good to me

**Comment by sapnasivakumar** [comment]
@hmhealey I've updated the snapshots. 
Can you please have a look when possible and let me know if I can do any further.

**Comment by hmhealey** [comment]
@lindy65 I guess the issue with the test server is probably something to do with this being a PR from a fork since I remember that being an issue at some point. I'm going to submit a PR with a copy of the branch so we can test on that

**Comment by lindy65** [comment]
> @lindy65 I guess the issue with the test server is probably something to do with this being a PR from a fork since I remember that being an issue at some point. I'm going to submit a PR with a copy of the branch so we can test on that
> 
> **Edit:** The test server will be on #26044 when it's created
> 
> **Second Edit:**
> 
> > Mattermost test server created! 🎉
> > Access here: https://mattermost-pr-26044.test.mattermost.cloud
> > Account Type 	Username 	Password
> > Admin 	sysadmin 	Sys@dmin123
> > User 	user-1 	User-1@123
> > Your Spinwick's installation ID is: `mfecippnjjnz7mn69yfjyf5nny` To access the logs, please click [here](https://grafana.internal.mattermost.com/explore?orgId=1&left=%7B%22datasource%22:%22PFB2D5CACEC34D62E%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnamespace%3D%5C%22mfecippnjjnz7mn69yfjyf5nny%5C%22%7D%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22PFB2D5CACEC34D62E%22%7D,%22editorMode%22:%22code%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D)

Thanks @hmhealey :)

**Review by lindy65 — APPROVED** [review]
Thanks @sapnasivakumar - looks good to me 👍 

cc// @hmhealey - not sure whether we're still waiting on `wiggin77` for a review or whether this is good to merge?
Also, good to close the PR you opened for the spinwick, thanks!

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:   Great work @sapnasivakumar

**Pull request merged**
